README.md

```diff
@@ -1,18 +1,18 @@
 opam-cross-windows
 ==================
 
-This repository contains an up-to-date Windows toolchain featuring OCaml `4.07.0`, `4.08.0`, `4.12.0`, `4.14.0` and `5.1.1`, as well as some commonly used packages.
+This repository contains an up-to-date Windows toolchain featuring OCaml `4.07.0`, `4.08.0`, `4.12.0`, `4.14.1`, `5.1.1` and `5.3.0`, as well as some commonly used packages.
 
 The supported build systems are 32-bit and 64-bit x86 Linux and `arm64` macos (Apple Silicon). The supported target systems are 32-bit and 64-bit x86 Windows.
 
-Only 64-bit build and target are supported for OCaml `5.1.1`.
+Only 64-bit build and target are supported now on (from OCaml `5.1.1`.
 
 If you need support for other platforms or versions, please [open an issue](https://github.com/ocaml-cross/opam-cross-windows/issues).
 
 Prerequisites
 -------------
 
-A C cross-compiler targeting the appropriate Windows platform must be installed. On Debian derivatives they are provided in the `gcc-mingw-w64-i686` (for 32-bit x86 Windows targets) or `gcc-mingw-w64-x86-64` (for 64-bit x86 Windows targets) packages. Alternatively, the [MXE environment](http://mxe.cc) can be used.
+A C cross-compiler targeting the appropriate Windows platform must be installed. On Debian derivatives they are provided in the `gcc-mingw-w64-x86-64` packages (`gcc-mingw-w64-i686` for 32-bit x86 Windows targets). Alternatively, the [MXE environment](http://mxe.cc) can be used.
```

Comment:
I think that we should remove all `x86` documentation and support but perhaps not for this PR.